deploy/counter-demo/demo-app.yaml

```diff
@@ -213,9 +213,9 @@ Resources:
           Image: !FindInMap [Config, Workload, Image]
           Environment:
             - Name: REDIS_HOST
-              Value: 127.0.0.1
+              Value: redis.mesh.local
```

Comment:
would be cool if we had two outbounds and we could test both this naming and `MeshService`